### PR TITLE
chore: Update pdf list and flush workflow

### PR DESCRIPTION
**Pull request by ran-huang**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: .github/workflows/flush.yaml [hunk 1]
```diff
@@ -23,4 +23,7 @@ jobs:
       - name: refresh url
         run: |
           qshell account ${{ secrets.QINIU_ACCESS_KEY }} ${{ secrets.QINIU_SECRET_KEY }} tidb
-          qshell cdnrefresh -i cdnfresh.txt
+          result=$(qshell cdnrefresh -i cdnfresh.txt | grep -w "200")
+          if [ -z "$result" ]; then
+            exit 1
+          fi
```
Comment (ran-huang) [inline .github/workflows/flush.yaml:29]: A success reponse is `[I]  CDN refresh Code: 200, FlowInfo: success`. So I set it to fail if `200` is not found.

File: .github/workflows/flush.yaml [hunk 2]
```diff
@@ -23,4 +23,7 @@ jobs:
       - name: refresh url
         run: |
           qshell account ${{ secrets.QINIU_ACCESS_KEY }} ${{ secrets.QINIU_SECRET_KEY }} tidb
-          qshell cdnrefresh -i cdnfresh.txt
+          result=$(qshell cdnrefresh -i cdnfresh.txt | grep -w "200")
+          if [ -z "$result" ]; then
+            exit 1
```
Comment (TomShawn) [inline .github/workflows/flush.yaml:28]:
```suggestion
            echo "status failed"
            exit 1
```

File: .github/workflows/flush.yaml (hunk 1 above)
Comment (Oreoxmt) [inline .github/workflows/flush.yaml:29]:
```suggestion
          if qshell cdnrefresh -i cdnfresh.txt | grep -w '200'; then
            echo 'CDN refresh success'
          else
            exit 1
          fi
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Review by TomShawn — APPROVED** [review]

**Pull request merged**